packages/@okta/vuepress-theme-prose/global-components/ApiLifecycle.vue

```diff
@@ -9,20 +9,42 @@
     <span class="api-label api-label-deprecated" v-if="access == 'deprecated'">
       <i class="fa fa-fire-extinguisher"></i> Deprecated
     </span>
+    <span class="api-label api-label-ie" v-if="access == 'ie'">
+      Identity Engine
+    </span>
   </SmartLink>
 </template>
 
 <script>
 export default {
   name: "ApiLifecycle",
   components: {
-    SmartLink: () => import("../components/SmartLink.vue"),
+    SmartLink: () => import("../components/SmartLink.vue")
   },
   props: {
     access: {
       type: String,
-      default: "beta",
-    },
+      default: "beta"
+    }
+  },
+  data() {
+    return {
+      labelType: {
+        access: "access",
+        erlyAccess: "ea",
+        deprecated: "deprecated",
+        indentityEngine: "ie"
+      },
+      defaultLink: "/docs/reference/releases-at-okta/",
+      ieLink: "/docs"
```

Comment:
Should these be set as `const DEFAULT_LINK = "/docs/reference/releases-at-okta/"` for example ?